docs/installation/manual/installing_solr.md

```diff
@@ -0,0 +1,99 @@
+
+```bash
+cd /opt/solr
+sudo mkdir -p /var/solr/data/SOLR_CORE/conf
+sudo cp -r example/files/conf/* /var/solr/data/SOLR_CORE/conf
+sudo chown -R solr:solr /var/solr
+sudo -u solr bin/solr create -c SOLR_CORE -p 8983
+```
+- `SOLR_CORE`: `islandora8`
+
+### Installing `search_api_solr`
+
+Rather than use an out-of-the-box configuration that won’t be suitable for our purposes, we’re going to use the Drupal `search_api_solr` module to generate one for us. This will also require us to install the module so we can create these configurations using Drush.
+
+```bash
+cd /opt/drupal
+sudo -u www-data composer require drupal/search_api_solr:^3.0
+drush -y en search_api_solr
+```
+
+### Configuring search_api_solr
+
+Before we can create configurations to use with Solr, the core we created earlier needs to be referenced in Drupal.
+
+Log in to the Drupal site at `/user` using the sitewide administrator username and password, then navigate to `/admin/config/search/search-api/add-server`.
+
+Fill out the server addition form using the following options:
+
+![Adding a Solr Search Server](../../assets/adding_a_solr_search_server.png)
+
+![Configuring the Standard Solr Connector](../../assets/configuring_standard_solr_connector.png)
+
+![Setting the Solr Install Directory](../../assets/setting_the_solr_install_directory.png)
+
+- `SERVER_NAME`: `islandora8`
+    - This is completely arbitrary, and is simply used to differentiate this search server configuration from all others. **Write down** or otherwise pay attention to the `machine_name` generated next to the server name you type in; this will be used in the next step.
+
+!!! notice
+    You can ignore the error about an incompatible Solr schema; we're going to set this up in the next step. In fact, if you refresh the page after restarting Solr in the next step, you should see the error disappear.
+
+### Generating and Applying Solr Configurations
+
+Now that our core is in place and our Drupal-side configurations exist, we’re ready to generate Solr configuration files to connect this site to our search engine.
+
+```bash
+cd /opt/drupal
+drush solr-gsc SERVER_MACHINE_NAME /opt/drupal/solrconfig.zip
+unzip -d ~/solrconfig solrconfig.zip
+sudo cp ~/solrconfig/* /var/solr/data/SOLR_CORE/conf
+sudo systemctl restart solr
+```
+- `SERVER_MACHINE_NAME`: This should be the `machine_name` that was automatically generated when creating the configuration in the above step.
+
+### Adding an Index
+
+In order for content to be indexed back into Solr, a search index needs to be added to our server. Navigate to `/admin/config/search/search-api/add-index` and check off the things you'd like to be indexed.
+
+![Adding a Search Index](../../assets/adding_a_search_index.png)
+
+![Specifying the Solr Server](../../assets/specifying_the_solr_server.png)
+
+Click **Save** to add your index and kick off indexing of existing items.
```

Comment:
Something went wrong with the configuration steps. Saving my configuration did make the original "incompatible Solr schema" message disappear; but saving the index configuration resulted in the following "Server index status" message:
```
Error while checking server index status: An error occurred while trying to search with Solr: { "error":{ "metadata":[ "error-class","org.apache.solr.core.SolrCoreInitializationException", "root-error-class","java.lang.ClassNotFoundException"], "msg":"SolrCore 'islandora8' is not available due to init failure: Could not load conf for core islandora8: Can't load schema /var/solr/data/islandora8/conf/schema.xml: Plugin init failure for [schema.xml] fieldType \"collated_en\": Error loading class 'solr.ICUCollationField'",
```
(trimmed for brevity)

Reply:
I *knew* this seemed familiar. The default value for `solr.install.dir` in the config files provided by solr-gsc is `../../..` which does *not* resolve to the solr install directory. You need to manually edit `/var/solr/data/SOLR_CORE/conf/solrcore.properties` so that `solr.install.dir=/opt/solr`. 

See [this comment on the search_api_solr issues queue](https://www.drupal.org/project/search_api_solr/issues/3015993#comment-13327416).

Reply:
AH! I see what happened. You have the solr.install.dir field configured correctly in the screen-shot for setting up the server config in Drupal, but I missed it. Perhaps some narrative text to point out the key fields?